<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>more_block</class>
 <widget class="QWidget" name="more_block">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoreBlock</string>
  </property>
  <widget class="QPushButton" name="connect_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="setup_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="agreement_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>535</y>
     <width>77</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
